src/js/components/sharedComponents/DownloadStaticFile.jsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * DownloadStaticFile.jsx
+ * Created by Brian Petway 01/12/23
+ */
+
+import React from 'react';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faFileDownload } from "@fortawesome/free-solid-svg-icons";
+
+const DownloadStaticFile = () => (
+    <div className="download_static-file__container">
+        <a
+            href="/data/data-sources-download.pdf"
```

Comment:
i think the path could be passed in as a separate prop.  that would pretty easily make the component reusable.